tests/utils/SparseGridTest.cpp

```diff
@@ -94,17 +107,25 @@ TEST_F(GetNearbyTest, getNearbyLine2)
     }
     const std::vector<typename SparsePointGridInclusive<Point>::Elem> result = grid.getNearby(target, grid_size);
 
-    //Are all near points reported as near?
+    // Are all near points reported as near?
     for (const Point point : near)
     {
-        EXPECT_NE(result.end(), std::find_if(result.begin(), result.end(), [&point](const typename SparsePointGridInclusive<Point>::Elem &elem) { return elem.val == point; }))
-            << "Point " << point << " is near " << target << " (distance " << vSize(point - target) << "), but getNearby didn't find it. Grid size: " << grid_size;
+        EXPECT_NE(result.end(),
+                  std::find_if(result.begin(),
+                               result.end(),
+                               [&point](const typename SparsePointGridInclusive<Point>::Elem& elem) { return elem.val == point; }))
+          << "Point " << point << " is near " << target << " (distance " << vSize(point - target)
+          << "), but getNearby didn't find it. Grid size: " << grid_size;
```

Comment:
Indent with 4 spaces.

I also think that the line can be broken up more logically. This is halfway through the print of a coordinate. Breaking it up makes it less readable unless it is logically clustered.